assopy/tests/factories/order.py

```diff
@@ -11,3 +17,25 @@ class Meta:
 
     payment = 'cc' # cc because stripe is a credit card
     items = []
+
+
+class CouponFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = 'assopy.Coupon'
+
+    conference = factory.Iterator(conference.models.Conference.objects.all())
```

Comment:
isn't that executed at load/run time (and we probably want it to execute it on "call time" [whatever proper terminology is here. :P])

Reply:
I copied it from another factory, it should probably just fetch the current conference instead :) Updating.